### DISCUSSION TITLE
windows_eventlog entries in SOC Alert overview

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#4511] [Unsupported Versions] [answered]
Hello!

I get Windows event logs shipped to SO via Wazuh, but I don't want them to trigger alerts in the SOC Alert  overview. How can I disable them? Thanks in advance!

Cheers, Ben

**Comment by facyber** [comment, marked as answer]
Hi @ben-sec 

As I was interested in this, I was following guidance from this https://github.com/Security-Onion-Solutions/securityonion/discussions/2771#discussioncomment-361583 and it helped me to achieve that, at least on test environment and with small number of test.

I will see to send pull request to update the doc with examples but here it is in short:

1. On node you are sending Wazuh logs edit `/opt/so/rules/hids/local_rules.xml` file:
2. Copy rule you want to edit from `/opt/so/rules/hids/ruleset/rules` file(s) into `local_rules.xml`
3. Make changes you want, or in case you only want to stop alert just add **noalert="1" to the `<rule>` head tag**
4. Restart Wazuh on node `so-wazuh-restart`
5. Your file should look something like bellow.

```
<group name="localrules,">
  <rule id="60106" level="3" noalert="1" overwrite="yes">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^528$|^540$|^673$|^4624$|^4769$</field>
    <description>Windows Logon Success</description>
    <options>no_full_log</options>
    <mitre>
      <id>T1078</id>
    </mitre>
    <group>authentication_success,pci_dss_10.2.5,gpg13_7.1,gpg13_7.2,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>
</group>
```

Hope this helps. :)

**Reply by ben-sec** [reply]
Hi facyber, so I have to do this on a per rule basis? Thanks for your help!

**Reply by facyber** [reply]
Hi @ben-sec 

It seems like that. If you have a lot of rules, I guess with some basic BASH script you can insert **noalert="1" overwrite="yes"** in any rule that is for example level 3 or something like that. 

No problem :)

**Reply by 0nnyx** [reply]
Another way is to ship win event logs with osquery or filebeat instead of wazuh, which won't generate any alert.

**Reply by facyber** [reply]
Yes, but in my experience Wazuh is much more stable. I tested multiple times with osquery and there were always some issues, either with certificate, or agent cannot talk with SO, some random agent issues. I really wanted to move to osquery but yea...

**Reply by ben-sec** [reply]
I switched to winlogbeat and it works fine. Thanks for the tip!
